lib/double_entry/reporting/aggregate.rb

```diff
@@ -93,16 +112,17 @@ def range_is_complete?
 
       def field_hash
         {
-          :function => function,
-          :account => account,
-          :code => code,
-          :year => range.year,
-          :month => range.month,
-          :week => range.week,
-          :day => range.day,
-          :hour => range.hour,
-          :filter => filter.inspect,
-          :range_type => range.range_type.to_s,
+          :function        => function,
```

Comment:
I actually prefer the hash rocket, unless it's for a keyword argument for a function!  I think the ruby styleguide also prefers it: https://github.com/styleguide/ruby